test/integration/targets/win_mssql_login/tasks/main.yml

```diff
@@ -0,0 +1,78 @@
+# test code for the win_psmodule module when using winrm connection
+# (c) 2017, Daniele Lazzari <lazzari@mailup.com>
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+- name: get facts
+  setup:
+
+- name: check sql server is installed
+  win_shell: Get-WmiObject win32_service|?{$_.name -like "MSSQL*"}
+  register: service
+
+- name: run tests
+  when: service.stdout != ''
+  block:
+
+    - name: install SqlServer powershell module
```

Comment:
If this and other .msi files listed below are needed to run this module, they should be mentioned in the module documentation (i think there is a 'requirements' section).
Also I would consider adding the steps to install these required things into the 'examples' section of the documentation page.